scripts/run_pylint.py

```diff
@@ -66,7 +66,7 @@
 }
 TEST_RC_REPLACEMENTS = {
     'FORMAT': {
-        'max-module-lines': 1825,
+        'max-module-lines': 1840,
```

Comment:
Why should we be fiddling these majyk numbers just to get pylint to pass, if they don't represent "real" opinions?  https://github.com/tseaver/gcloud-python/commit/a43640dfb88db0955aac00f1614f2dff00cb7fa7 just turns off `too-many-lines` (and `too-many-statements`) for test code altogether, which I find preferable.